### PR TITLE
Homework5: Valeria Kudelko

### PR DESCRIPTION
**Pull request by ValeriaKudelko**

### CODE REVIEW

File: firstfile.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+k[0], k[1] = k[1], k[0]
+new_k = " ".join(k)
+print(new_k)
+
+             
+#!4.Напишите программу которая удаляет пробел в начале, в конце строки.
+
+pul = " one two three "
+print(pul.lstrip())
+print(pul.rstrip())
```
Comment (Stml89) [inline firstfile.py]: + print(pul.strip())

File: firstfile.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!1.Заменить символ “#” на символ “/” в строке 'www.my_site.com#about'.
+
+a = 'www.my_site.com#about'
+b = a.replace('#','/')
+print(b)
```
Comment (Stml89) [inline firstfile.py]: You do not need extra variable `b`

a = 'www.my_site.com#about'.replace('#','/')
print(a)

File: firstfile.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+d = s +'ing', v + 'ing'
+print(d)
+
+
+#!3.В строке “Ivanou Ivan” поменяйте местами слова => "Ivan Ivanou".
+
+k = "Ivanou Ivan".split()
+k[0], k[1] = k[1], k[0]
+new_k = " ".join(k)
+print(new_k)
```
Comment (Stml89) [inline firstfile.py]: k = "Ivanou Ivan".split()\
new_k = " ".join([k[1], k[0]])
print(new_k)

**Review by Stml89 — CHANGES_REQUESTED** [review]
Please fix linters!

File: firstfile.py (hunk 1 above)
Comment (ValeriaKudelko) [reply firstfile.py]: corrected

File: firstfile.py (hunk 2 above)
Comment (ValeriaKudelko) [reply firstfile.py]: corrected

File: firstfile.py (hunk 3 above)
Comment (ValeriaKudelko) [reply firstfile.py]: corrected

**Comment by Stml89** [comment]
pylinter still complaining about errors. we have discussed this issue in the discord https://discord.com/channels/1228753853046984774/1228754608101265449/1238773811932037150

**Review by Stml89 — APPROVED** [review]

**Pull request merged**